From b52f842b7182863ba9bde5923d59df75d027a63d Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui>
Date: Fri, 22 Dec 2023 11:35:19 +0800
Subject: [PATCH] Makefile.runner_helper.mk

---
 Makefile.runner_helper.mk | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/Makefile.runner_helper.mk b/Makefile.runner_helper.mk
index 7b61f4fb4..e351c7b87 100644
--- a/Makefile.runner_helper.mk
+++ b/Makefile.runner_helper.mk
@@ -15,11 +15,7 @@ TAR_XZ_ARGS ?= -f -0
 # Tar files that we want to generate from the Docker file system, this is
 # generally used for linux based Dockerfiles.
 BASE_TAR_PATH := out/helper-images/prebuilt
-TAR_XZ += ${BASE_TAR_PATH}-x86_64.tar.xz
-TAR_XZ += ${BASE_TAR_PATH}-x86_64-pwsh.tar.xz
-TAR_XZ += ${BASE_TAR_PATH}-arm.tar.xz
-TAR_XZ += ${BASE_TAR_PATH}-arm64.tar.xz
-TAR_XZ += ${BASE_TAR_PATH}-s390x.tar.xz
+TAR_XZ += ${BASE_TAR_PATH}-loongarch64.tar.xz
 
 # Binaries that we support for the helper image. We are using the following
 # pattern match:
@@ -27,11 +23,7 @@ TAR_XZ += ${BASE_TAR_PATH}-s390x.tar.xz
 # match up with GO_ARCH_* variables names. Note that Linux is implied by
 # default.
 BASE_BINARY_PATH := out/binaries/gitlab-runner-helper/gitlab-runner-helper
-BINARIES := ${BASE_BINARY_PATH}.x86_64-windows
-BINARIES += ${BASE_BINARY_PATH}.x86_64
-BINARIES += ${BASE_BINARY_PATH}.arm
-BINARIES += ${BASE_BINARY_PATH}.arm64
-BINARIES += ${BASE_BINARY_PATH}.s390x
+BINARIES := ${BASE_BINARY_PATH}.loongarch64
 
 # Define variables with the architecture for each matching binary. We are using
 # the following pattern match GO_ARCH_{{arch}}-{{os}}, these should match up
@@ -40,6 +32,7 @@ GO_ARCH_x86_64 = linux/amd64
 GO_ARCH_arm = linux/arm
 GO_ARCH_arm64 = linux/arm64
 GO_ARCH_s390x = linux/s390x
+GO_ARCH_loongarch64 = linux/loong64
 GO_ARCH_x86_64-windows = windows/amd64
 
 # Go files that are used to create the helper binary.
-- 
2.27.0

