From 2121280f3fd673e3c8a53c7506856015adcb1f06 Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui>
Date: Fri, 22 Dec 2023 11:34:36 +0800
Subject: [PATCH] Makefile

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 9170af2ed..47c94e394 100644
--- a/Makefile
+++ b/Makefile
@@ -14,8 +14,8 @@ endif
 
 PACKAGE_CLOUD ?= ayufan/gitlab-ci-multi-runner
 PACKAGE_CLOUD_URL ?= https://packagecloud.io/
-BUILD_ARCHS ?= -arch '386' -arch 'arm' -arch 'amd64' -arch 'arm64' -arch 's390x'
-BUILD_PLATFORMS ?= -osarch 'darwin/amd64' -os 'linux' -os 'freebsd' -os 'windows' ${BUILD_ARCHS}
+BUILD_ARCHS ?= -arch 'loong64'
+BUILD_PLATFORMS ?= -osarch 'linux/loong64'
 S3_UPLOAD_PATH ?= master
 
 # Keep in sync with docs/install/linux-repository.md
@@ -360,7 +360,7 @@ $(GOCOVER_COBERTURA):
 	go get github.com/boumenot/gocover-cobertura
 
 $(GOX):
-	go get github.com/mitchellh/gox
+	@echo "skip download gox"
 
 $(GOLANGLINT): TOOL_BUILD_DIR := .tmp/build/golangci-lint
 $(GOLANGLINT): $(GOLANGLINT_GOARGS)
-- 
2.27.0

