From 8986b610082d917da8459059f20bebc2da7f074d Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui>
Date: Fri, 22 Dec 2023 11:34:54 +0800
Subject: [PATCH] Makefile.build.mk

---
 Makefile.build.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.build.mk b/Makefile.build.mk
index 39a4f490c..b471ef01e 100644
--- a/Makefile.build.mk
+++ b/Makefile.build.mk
@@ -6,7 +6,7 @@ runner-bin: $(GOX)
 		   $(PKG)
 
 runner-bin-host: OS := $(shell uname -s | tr '[:upper:]' '[:lower:]')
-runner-bin-host: ARCH := $(shell uname -m | sed s/x86_64/amd64/ | sed s/i386/386/)
+runner-bin-host: ARCH := $(shell uname -m | sed s/x86_64/amd64/ | sed s/i386/386/ | sed s/loongarch64/loong64/)
 runner-bin-host:
 	# Building $(NAME) in version $(VERSION) for host platform
 	$(MAKE) runner-bin BUILD_PLATFORMS="-osarch=$(OS)/$(ARCH)"
-- 
2.27.0

