From 418576019d2d8201c0c1d956cd405b6a91d134e4 Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui@loongson.cn>
Date: Wed, 27 Dec 2023 10:48:51 +0800
Subject: [PATCH] Makefile

---
 Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index f0962022..5950df5a 100644
--- a/Makefile
+++ b/Makefile
@@ -173,6 +173,7 @@ BUILD_TARGETS = \
 	bin/git-lfs-linux-amd64 \
 	bin/git-lfs-linux-ppc64le \
 	bin/git-lfs-linux-s390x \
+	bin/git-lfs-linux-loong64 \
 	bin/git-lfs-linux-386 \
 	bin/git-lfs-freebsd-amd64 \
 	bin/git-lfs-freebsd-386 \
@@ -215,6 +216,8 @@ bin/git-lfs-linux-ppc64le : $(SOURCES) mangen
 	$(call BUILD,linux,ppc64le,-linux-ppc64le)
 bin/git-lfs-linux-s390x : $(SOURCES) mangen
 	$(call BUILD,linux,s390x,-linux-s390x)
+bin/git-lfs-linux-loong64 : $(SOURCES) mangen
+	$(call BUILD,linux,loong64,-linux-loong64)
 bin/git-lfs-linux-386 : $(SOURCES) mangen
 	$(call BUILD,linux,386,-linux-386)
 bin/git-lfs-freebsd-amd64 : $(SOURCES) mangen
@@ -274,6 +277,7 @@ RELEASE_TARGETS = \
 	bin/releases/git-lfs-linux-amd64-$(VERSION).tar.gz \
 	bin/releases/git-lfs-linux-ppc64le-$(VERSION).tar.gz \
 	bin/releases/git-lfs-linux-s390x-$(VERSION).tar.gz \
+	bin/releases/git-lfs-linux-loong64-$(VERSION).tar.gz \
 	bin/releases/git-lfs-linux-386-$(VERSION).tar.gz \
 	bin/releases/git-lfs-freebsd-amd64-$(VERSION).tar.gz \
 	bin/releases/git-lfs-freebsd-386-$(VERSION).tar.gz \
-- 
2.27.0

