From 733a7ea1c5b0b7fd2dabb1a9ad32303cc6fac668 Mon Sep 17 00:00:00 2001
From: Jingyun Hua <huajingyun@loongson.cn>
Date: Mon, 8 May 2023 09:01:19 +0800
Subject: [PATCH] TDengine ver-3.0.0.0 taos-tools add loongarch

---
 src/CMakeLists.txt | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8aef9ba..9dddb6d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -25,6 +25,9 @@ ELSEIF (CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
     SET(CURRENT_CPUTYPE "aarch64")
     MESSAGE(STATUS "Set CPUTYPE to aarch64")
     SET(CURRENT_PLATFORM_ARCH_STR "arm64")
+ELSEIF (CMAKE_SYSTEM_PROCESSOR MATCHES "loongarch64")
+    SET(CURRENT_CPUTYPE "loongarch64")
+    MESSAGE(STATUS "Set CPUTYPE to loongarch64")
 ELSEIF (CMAKE_SYSTEM_PROCESSOR MATCHES "mips64")
     SET(CURRENT_CPUTYPE "mips64")
     MESSAGE(STATUS "Set CPUTYPE to mips64")
@@ -163,7 +166,7 @@ IF (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin
             "${CMAKE_BINARY_DIR}/build/lib/libjansson.a")
 
         ADD_LIBRARY(snappy STATIC IMPORTED)
-        IF(${OS_ID} MATCHES "(CentOS)|(centos)" OR ${OS_ID} MATCHES "kylin"
+        IF(${OS_ID} MATCHES "(CentOS)|(centos|loongnix-server)" OR ${OS_ID} MATCHES "kylin"
                 OR ${OS_ID} MATCHES "rhel" OR ${OS_ID} MATCHES "rocky")
             SET_PROPERTY(TARGET snappy PROPERTY IMPORTED_LOCATION
                 "${CMAKE_BINARY_DIR}/build/lib64/libsnappy.a")
@@ -285,7 +288,7 @@ IF (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin
             MESSAGE(STATUS "src cmake LN303, detected The current OS is " ${OS_ID} "")
 
             ADD_LIBRARY(avro STATIC IMPORTED)
-            IF(${OS_ID} MATCHES "(CentOS)|(centos)" OR ${OS_ID} MATCHES "kylin"
+            IF(${OS_ID} MATCHES "(CentOS)|(centos|loongnix-server)" OR ${OS_ID} MATCHES "kylin"
                     OR ${OS_ID} MATCHES "rhel" OR ${OS_ID} MATCHES "rocky")
                 SET_PROPERTY(TARGET avro PROPERTY IMPORTED_LOCATION
                     "${CMAKE_BINARY_DIR}/build/lib64/libavro.a")
-- 
2.27.0

