From a0c889a1021705d5255d37345731204289afcace Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui@loongson.cn>
Date: Wed, 27 Dec 2023 17:44:39 +0800
Subject: [PATCH] go.mod

---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index d5b7198..efd1fa0 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/mitchellh/gox
+module github.com/Loongson-Cloud-Community/gox
 
 go 1.17
 
-- 
2.27.0

