From f8340fcaf0cc394ab73ba294ea6aef2e6d8295d6 Mon Sep 17 00:00:00 2001
From: qiangxuhui <qiangxuhui@loongson.cn>
Date: Wed, 27 Dec 2023 17:43:28 +0800
Subject: [PATCH] platform.go

---
 platform.go | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/platform.go b/platform.go
index 3d4ab71..e3f5639 100644
--- a/platform.go
+++ b/platform.go
@@ -178,7 +178,13 @@ var (
 	// no new platforms in 1.18
 	Platforms_1_18 = Platforms_1_17
 
-	PlatformsLatest = Platforms_1_18
+	Platforms_1_19 = addDrop(Platforms_1_18, []Platform{
+		{"linux", "loong64", true},
+	}, nil)
+
+	Platforms_1_20 = Platforms_1_19
+
+	PlatformsLatest = Platforms_1_20
 )
 
 // SupportedPlatforms returns the full list of supported platforms for
@@ -221,6 +227,8 @@ func SupportedPlatforms(v string) []Platform {
 		{">= 1.16, < 1.17", Platforms_1_16},
 		{">= 1.17, < 1.18", Platforms_1_17},
 		{">= 1.18, < 1.19", Platforms_1_18},
+		{">= 1.19, < 1.20", Platforms_1_19},
+		{">= 1.20, < 1.21", Platforms_1_20},
 	}
 
 	for _, p := range platforms {
-- 
2.27.0

